Problem setting it up

```
$ NODE_PATH=./node_modules node scraper.js import

module.js:340
    throw err;
          ^
Error: Cannot find module './31C3/scraper.js'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/mjob/Projekte/re-data/scraper/config/scrapers.js:5:19)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
```

My `./config.js`:

```
#!/usr/bin/env node

/* configuration */
module.exports = {

    /* listen on tcp or socket */
    app: {
        host: 'localhost',
        port: 9999
    },

    /* couchdb configuration */
    db: {
        database: 'rp-data',
        host: 'localhost',
        port: 5984,
        options: {
            secure: false,
            auth: {
                username: '',
                password: ''
            }
        }
    },

    /* version */
    version: 0.1

};
```

my `scrapers/config/scraper.js`

```
exports.scrapers = {
    // { module:require('./rp13/scraper.js'), db:true },
    // { module:require('./rp14/scraper.js'), db:true }
    // { module:require('./altconf14/scraper.js'), db:false }
    '30C3': { module:require('./31C3/scraper.js'), db:true }
}
```

cc @toto - feel free to ping me on Twitter

--- a/readme.md
+++ b/readme.md
@@ -18,3 +18,17 @@ The infrastructure is still very rough as this is a side project of several peop
 - An app for [AltConf](https://itunes.apple.com/us/app/id881934035) based on one of the re-publica apps.
 - [Congress – 31C3](https://itunes.apple.com/de/app/congress-31c3/id941205524) for the [31st Chaos Communication Congress](https://events.ccc.de/congress/2014)
 - More…
+
+## Set it up locally
+
+ * you need a CouchDB instance (use for example a docker container to set on up easily)
+   * `docker run -d -p 5984:5984 fedora/couchdb`
+   * `curl -X PUT http://localhost:5984/_config/admins/user -d '"secret"'` (creates user `user` with password `secret`)
+ * copy `config.js.dist` to `config.js` and fill in the credentials for the CouchDB instance (see curl step above)
+ * copy `scraper/config/scrapers.js.example` to `scraper/config/scrapers.js` (default config is fine for a first run)
+ * fetch dependencies via `npm` (needs to be executed in `scraper` subdirectory):
+   * `npm install`
+ * run the `resetDB` command inside the `scraper` subdirectory:
+   * `NODE_PATH=node_modules node scraper.js resetDB` (`NODE_PATH` just specifies not globally install locations - was created by the `npm install` step)
+ * run the `import` command inside the `scraper` subdirectory:
+   * `NODE_PATH=node_modules node scraper.js import`

